halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/kubernetes/KubernetesAddAccountCommand.java

```diff
@@ -120,6 +120,12 @@ protected String getProviderName() {
       description = KubernetesCommandProperties.CACHE_THREADS)
   private int cacheThreads = 1;
 
+  @Parameter(
+      names = "--cache-interval-seconds",
+      arity = 1,
+      description = KubernetesCommandProperties.CACHE_INTERVAL_SECONDS_DESCRIPTION)
+  private Long cacheIntervalSeconds = 30L;
```

Comment:
This generally looks ok, but my reflex is to remove the default value.  It's in multiple places in the code here, which doesn't feel great.  If a default makes sense, I'd like to see it in clouddriver itself, e.g. [here](https://github.com/spinnaker/clouddriver/blob/1c53e6d25d33f23f0903ab8424440c2e0e753235/clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/config/KubernetesConfigurationProperties.java#L58).

Reply:
Thank you @dbyron-sf, Removed the default value for the parameter.

Reply:
@dbyron-sf , there is a slight issue with moving the default value to another repo, the default value won't be reflected in the docs as the scripts don't currently look into external repo for the default values. If that is ok, there should not be a problem

Reply:
I think we're OK relying on clouddriver's behavior when there's no cacheIntervalSeconds specified since that's where we are today, yes?